src/web/handlers/GestureHandler.ts

```diff
@@ -589,6 +587,10 @@ export default abstract class GestureHandler implements IGestureHandler {
     this.config = { enabled: enabled, ...props };
     this.enabled = enabled;
 
+    if (this.delegate.wasInitialized()) {
+      this.delegate.onEnabledChange(enabled);
+    }
```

Comment:
I think we can move this check inside the delegate, no?

1. It would allow for reactions even if delegate was not initialized yet
2. It keeps the logic outside of `GestureHandler`

Reply:
Done in [d80a9eb](https://github.com/software-mansion/react-native-gesture-handler/pull/3041/commits/d80a9eb3381db07b9367ba68cde87689b11ef7d1)